src/binding.ts

```diff
@@ -4,61 +4,195 @@
  * sonarlint@sonarsource.com
  * Licensed under the LGPLv3 License. See LICENSE.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
-'use strict';
```

Comment:
I think we should keep it for now (and maybe switch to `alwaysStrict` in our tsconfig which would allow us to remove it from all source files).